docs/src/main/sphinx/release/release-389.md

```diff
@@ -0,0 +1,40 @@
+# Release 389 (6 July 2022)
+
+## General
+
+* Improve performance of queries which contain a large number of aggregations
+  that use `RowType` for representing intermediate state (e.g. `SUM`). ({issue}`12762`)
+* Remove deprecated version of `ConnectorRecordSetProvider#getRecordSet`. ({issue}`13084`)
+* Fix query failure when no arguments are passed to a table function. ({issue}`12951`)
+* Fix error where a split keeps running after query failure due to an
+  uninterruptible regular expression function. ({issue}`13064`)
```

Comment:
@tangjiangling is this acceptable? I don't know how necessary it is to strictly mention JONI

Reply:
I think we should mention JONI, because the issue usually occurs when using the JONI regular library (which can take exponential time for certain regular expressions)